Cut to Candidate (09/23)

Jonasslv opened the pull request.

InvokerMaster committed:
filter by claimable (#252)

Co-authored-by: Techlord <dev@techlord.net>

InvokerMaster committed:
sort by claimable (#251)

Co-authored-by: Techlord <dev@techlord.net>

InvokerMaster committed:
implement withdraw-modal (#254)

fix withdraw-modal bugs

open withdraw_modal bug fixed & toast.dismiss()

fixed disabled logic

Co-authored-by: Techlord <dev@techlord.net>

Commit:
fixing undefined

fixed withdraw of deprecated pools

frostbournesb committed:
Merge pull request #259 from Snowball-Finance/task/jonas/fix-undefined

Fixing undefined bug

InvokerMaster committed:
update staking page when switch wallets (#262)

Co-authored-by: Techlord <dev@techlord.net>

InvokerMaster committed:
check claimable amount (#264)

check claim before trasacting

follow syntax

syntax

Co-authored-by: Techlord <dev@techlord.net>

InvokerMaster committed:
[DEV-158] Fix filter bugs (#231)

show my pool first

Update connectors.js

deporated pools show on the ending of the user pools list

update filter-bugs

all filters clear when account changed

reset state

Co-authored-by: Techlord <dev@techlord.net>

prodvdc committed:
updated img tags using next/image library, fixed compound and earn pair icons loading issue (#260)

fixed image issue on compound and earn page

Commit:
change healthy check json

frostbournesb committed:
Merge pull request #265 from Snowball-Finance/task/jonas/fix-healthy-response

change healthy check json

prodvdc committed:
close sidebar when clicking submenu item on mobile view (#266)

Commit:
cleanup

frostbournesb committed:
Merge pull request #267 from Snowball-Finance/task/jonas/somecleanup

Cleanup Code

prodvdc committed:
updated styling on underlying tokens line on Compound and Earn page (#269)

Sanjeev1308 committed:
Worked on the left over claimable snob (#271)

Sanjeev1308 committed:
Worked on the dark mode (#270)

* Worked on the dark mode

* Resolved the issue on dark mode

* Worked on the select box issue

* Worked on the governance banner issue

prodvdc commented:
LGTM.
I see some inconsistency in the coding style.
e.g. trailingComma, quote (single or double), semi, etc.
We can update it later by confirming rules.

ghost commented:
agree

prodvdc committed:
Fix/dark mode (#272)

* updated theme switch button for UX(ThemeButton)

* refactored and updated TopAppBar for mobile view

* fixed menu item background on dark mode

InvokerMaster committed:
check claimable amount (#263)

Call Claim when Withdrawing

check claim amount > 0

await claim

call before withdraw

set istransacting before claim

add one step in withdraw-modal

corrected some bugs

Co-authored-by: Techlord <dev@techlord.net>

InvokerMaster committed:
show claim btn when the card is collapsed (#253)

show claim btn when no LP

Show Claim btn in getProperAction

fixed some bugs

Co-authored-by: Techlord <dev@techlord.net>

prodvdc commented on contexts/provider-context.js:
```diff
@@ -15,63 +15,66 @@ export function ProviderProvider({ children }) {
-    var raw = JSON.stringify({"jsonrpc":"2.0","id":1,"method":"health.getLiveness"});
-  
+
+    var raw = JSON.stringify({ "jsonrpc": "2.0", "id": 1, "method": "health.getLiveness" });
```
const raw = JSON.stringify({ "jsonrpc": "2.0", "id": 1, "method": "health.getLiveness" });

Jonasslv replied:
honestly we need to review all our code for this kind of thing, this will not change the result of the function for now, so I will discard this and we make a cleanup task later

prodvdc commented on contexts/provider-context.js:
```diff
@@ -15,63 +15,66 @@ export function ProviderProvider({ children }) {
   const nodeIsHealthy = async (url) => {
     var myHeaders = new Headers();
```
const myHeaders = new Headers();

Jonasslv replied:
honestly we need to review all our code for this kind of thing, this will not change the result of the function for now, so I will discard this and we make a cleanup task later

prodvdc commented on contexts/provider-context.js:
```diff
@@ -15,63 +15,66 @@ export function ProviderProvider({ children }) {
+
+    var raw = JSON.stringify({ "jsonrpc": "2.0", "id": 1, "method": "health.getLiveness" });
+
     var requestOptions = {
```
const requestOptions = {

Jonasslv replied:
honestly we need to review all our code for this kind of thing, this will not change the result of the function for now, so I will discard this and we make a cleanup task later

prodvdc commented on parts/Compound/CompoundDialogs/index.js:
```diff
@@ -177,7 +206,7 @@ const CompoundDialogs = ({
         </Grid>
       </div>
-      <SnowStepBox transactionStatus={transactionStatus}/>
+      {<SnowStepBox transactionStatus={transactionStatus} title={title}/>}
```
<SnowStepBox transactionStatus={transactionStatus} title={title}/>

Jonasslv replied:
we can add that to a cleanup task later, the SnowStepBox DOM already has treatment for this
